truss-train/truss_train/log_utils.py

```diff
@@ -9,10 +9,15 @@
 
 from truss.remote.baseten.api import BasetenApi
 
-POLL_INTERVAL_SEC = 5
+POLL_INTERVAL_SEC = 2
```

Comment:
Unrelated, but I think our BE can handle polling every 2s, and it leads to a snappier tail experience